src/D2L.Bmx/D2L.Bmx.csproj

```diff
@@ -8,6 +8,7 @@
     <InvariantGlobalization>true</InvariantGlobalization>
     <RootNamespace>D2L.Bmx</RootNamespace>
     <LangVersion>preview</LangVersion>
+    <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
```

Comment:
`NtQueryInformationProcess` needs this

Reply:
It's really LibraryImport that needs this, but you aren't using LibraryImport yet. Any reason not to?

Reply:
Oh right. I was getting some errors when trying to build and it was from using library import. Let me copy them into a slack thread because I may need some help

Reply:
Sorry no I did manage to fix it but did not swap it back from DllImport to LibraryImport